tools/functional-tester/etcd-tester/tester.go

```diff
@@ -115,3 +121,77 @@ func (s *Status) setCase(c int) {
 	defer s.mu.Unlock()
 	s.Case = c
 }
+
+// checkConsistency stops the cluster for a moment and get the hashes of KV storages.
+// TODO: instead of logging to stdout, record it as an event to database.
+func (c *cluster) checkConsistency() error {
```

Comment:
i feel this function should return error if the result is inconsistency. and we should not print out the result in this func.